ui/components/GHImport.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react'
+import { Tooltip, IconButton, TextField, Button, Grid, Typography } from '@material-ui/core';
+import { makeStyles, MuiThemeProvider } from '@material-ui/core/styles';
+import { createTheme } from '@material-ui/core/styles';
+import GitHubIcon from '@material-ui/icons/GitHub';
+import FormControl from '@material-ui/core/FormControl';
+import Checkbox from '@material-ui/core/Checkbox';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import { URLValidator } from '../utils/URLValidator';
+
+const getMuiTheme = () => createTheme({
+  palette : {
+    primary : {
+      main : "#607d8b"
+    }
+  },
+  overrides : {
+    MuiGrid : {
+      input : {
+        color : '#607d8b'
+      }
+    },
+  }
+})
```

Comment:
I won't recommend using it. 
you have already used makeStyles and the same can be achieved with that.

Reply:
Oh yes, I was focussing on it's working much, Thanks for reminding, I'll clean the code styles using makeStyles before final commit.